<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatistics</class>
 <widget class="QDialog" name="DialogStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelPeriod">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Total hours</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Over Time</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogStatistics</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogStatistics</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DialogStatistics</receiver>
   <slot>details(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>details(QModelIndex)</slot>
  <slot>ownChanged(QString)</slot>
 </slots>
</ui>
